pulp_2_tests/tests/rpm/api_v2/test_modularity.py

```diff
@@ -200,3 +260,68 @@ def test_all(self):
             with self.subTest(package=key):
                 module = [line for line in lines if key in line]
                 self.assertEqual(len(module), value, module)
+
+
+class UploadModuleTestCase(unittest.TestCase):
+    """Upload a module.yaml file and test upload import in Pulp repo."""
+
+    def test_all(self):
+        """Verify whether uploaded module.yaml is reflected in the pulp repo."""
+        cfg = config.get_config()
+        if cfg.pulp_version < Version('2.17'):
+            raise unittest.SkipTest('This test requires at least Pulp 2.17 or newer.')
+        if not os_support_modularity(cfg):
+            raise unittest.SkipTest('This test requires an OS that supports modularity.')
+        client = api.Client(cfg, api.json_handler)
+        # Create a normal Repo without any data.
+        body = gen_repo(
+            importer_config={'feed': RPM_SIGNED_FEED_URL},
```

Comment:
I would suggest to use the `RPM_UNSIGNED_FEED_URL`. Just use the `RPM_SIGNED_FEED_URL` when it is a test requirement.